metagraph/src/graph/representation/succinct/boss_chunk_construct.cpp

```diff
@@ -404,12 +409,17 @@ void skip_same_suffix(const KMER &el, Decoder &decoder, size_t suf) {
 }
 
 /**
- * Generates non-redundant dummy-1 source k-mers and dummy sink kmers from #kmers.
- * @return a triplet containing the names of the original k-mer blocks, the dummy-1 source
- * k-mer blocks and the dummy sink k-mers
+ * Generates ordered blocks of k-mers for the BOSS table.
+ * Adds non-redundant dummy-1 source k-mers and dummy sink k-mers.
```

Comment:
I'm assuming "dummy-1 source k-mers" means every dummy k-mer except $$$$$$..., right?

Reply:
no, only with a single sentinel in the beginning: $AAAAAAA

Reply:
ah, right..never mind